### PR TITLE
[CLOUD-255] Docs/cloud 255 support oci repos

### PR DESCRIPTION
**Pull request by lreciomelero**
[Doc] Support for OCI Helm Repos

## Description
Information on how the OCI helm repositories are supported and how to use them has been added to this PR.

## Additional Information
- None

## Checklist
- [x] [PR desc] List any pull-request related to this change.
- [X] [PR labels] Add the corresponding labels (release, skips, cherry-pick, AT-eks-smoke, etc).
- [x] [Changelog] Add the changes to CHANGELOG.md.    
- [X] [Docs] Add the changes to the documentation.
- [X] [QA] Add the new unit tests according to the changes.

### CODE REVIEW

**Comment by stratiocommit** [comment]
This PR has modified files that have not been added for all versions. Please add them including the text [to be translated] so that the reviewer can easily upload or modify the translated versions. Missing folder/files:
EN - modules/ROOT/pages/architecture.adoc
EN - modules/ROOT/pages/installation.adoc

**Review by iamjanr — APPROVED** [review]
LGTM

File: stratio-docs/es/modules/ROOT/pages/architecture.adoc [hunk 1]
```diff
@@ -536,4 +536,28 @@ La siguiente tabla muestra los _registries_ soportados según proveedor/_flavour
...
+^|generic
+
+^|Azure
+^|generic
+
+^|AKS
+^|generic
+|===
+
+NOTE: Las url de los repositorios de tipo OCI llevan el prefijo *oci://*. Por ejemplo, oci://stratioregistry.azurecr.io/helm-repository-example
```
Comment (stg-0) [inline stratio-docs/es/modules/ROOT/pages/architecture.adoc:563]:
```suggestion
|===

== Helm Repository

Como prerrequisito de la instalación se debe indicar un repositorio de helm del que se pueda extraer el chart de _Cluster Operator_. Este repositorio puede utilizar protocolo https u oci (utilizado para repositorios de _cloud providers_ como ECR, GAR o ACR).

[.center,cols="2,1",width=40%]
|===
^|AWS
^|ecr, generic

^|EKS
^|ecr, generic

^|GCP
^|gar, gcr, generic

^|Azure
^|acr, generic

^|AKS
^|acr, generic
|===

NOTE: Las url de los repositorios de tipo OCI llevan el prefijo *oci://*. Por ejemplo, oci://stratioregistry.azurecr.io/helm-repository-example.

NOTE: Verificar en la documentación de keos-installer los repositorios que se soporten en la versión a utilizar.
```

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 2]
```diff
@@ -248,6 +248,34 @@ Los siguientes campos son considerados secretos del aprovisionamiento:
...
+
+|_type_ (*__string__*)
+|tipo del repositorio
+|generic o ecr
+|generic
+
+|===
+
+NOTE: Los repositorios ecr nunca son autenticados. La autenticación se realizará mediante las credenciales contenidas en el secrets.yml, a través del sdk de golang. Este tipo de repositorio únicamente está soportado para instalaciones sobre AWS y EKS.
+
```
Comment (stg-0) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:278]:
```suggestion
=== Helm Repository

Como prerrequisito de instalación, se debe indicar el repositorio helm del que se descargará el chart de helm del _Cluster Operator_. Este apartado permite indicar la url del repositorio, su tipo y si se trata de un repositorio autenticado.

[cols="1,4,2,1"]
|===
^|Nombre ^|Descripción ^|Ejemplo ^|Opcional

|_auth++_++required_
|Indica si el repositorio es autenticado
|false
|Si, default: false.

|_url_
|Url del repositorio
|*Repositorios OCI*: oci://stratioregistry.azurecr.io/helm-repository-example + 
*Repositorios HTTPS*: https://[IP]:8080
|No

|_type_
|Tipo del repositorio
|generic o ecr
|Si, default: generic

|===

NOTE: Los repositorios OCI (de _cloud providers_ como ECR, GAR o ACR) nunca son autenticados. La autenticación se realizará mediante las credenciales utilizadas en el aprovisionamiento. Verificar en la documentación de keos-installer los repositorios que se soporten en la versión a utilizar.

```

**Comment by stratiocommit** [comment]
This PR has modified files that have not been added for all versions. Please add them including the text [to be translated] so that the reviewer can easily upload or modify the translated versions. Missing folder/files:
EN - modules/ROOT/pages/architecture.adoc
EN - modules/ROOT/pages/installation.adoc

**Review by stg-0 — APPROVED** [review]
lgtm

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 3]
```diff
@@ -248,6 +248,34 @@ Los siguientes campos son considerados secretos del aprovisionamiento:
...
+Como prerrequisito de instalación, se debe indicar el repositorio helm del que se descargará el chart de helm del _Cluster Operator_. Este apartado permite indicar la url del repositorio, su tipo y si se trata de un repositorio autenticado.
+
+[cols="1,4,2,1"]
+|===
+^|Nombre ^|Descripción ^|Ejemplo ^|Opcional
+
+|_auth++_++required_
+|Indica si el repositorio es autenticado
+|false
+|Si, default: false.
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:262]:
```suggestion
| _auth++_++required_
| Indica si el repositorio es autenticado.
| _false_
| Sí. Por defecto: _false_.
```

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 4]
```diff
@@ -248,6 +248,34 @@ Los siguientes campos son considerados secretos del aprovisionamiento:
...
+
+|_url_
+|Url del repositorio
+|*Repositorios OCI*: oci://stratioregistry.azurecr.io/helm-repository-example + 
+*Repositorios HTTPS*: https://[IP]:8080
+|No
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:268]:
```suggestion
| _url_
| URL del repositorio.
| *Repositorios OCI*: oci://stratioregistry.azurecr.io/helm-repository-example + 
*Repositorios HTTPS*: https://[IP]:8080
| No
```

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 5]
```diff
@@ -248,6 +248,34 @@ Los siguientes campos son considerados secretos del aprovisionamiento:
...
+
+|_type_
+|Tipo del repositorio
+|generic o ecr
+|Si, default: generic
+
+|===
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:275]:
```suggestion
| _type_
| Tipo del repositorio.
| _generic_ o ecr.
| Sí. Por defecto: _generic_.
|===
```

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 6]
```diff
@@ -248,6 +248,34 @@ Los siguientes campos son considerados secretos del aprovisionamiento:
...
+
+NOTE: Los repositorios OCI (de _cloud providers_ como ECR, GAR o ACR) nunca son autenticados. La autenticación se realizará mediante las credenciales utilizadas en el aprovisionamiento. Verificar en la documentación de keos-installer los repositorios que se soporten en la versión a utilizar.
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:277]:
```suggestion
NOTE: Los repositorios OCI (de proveedores _cloud_ como ECR, GAR o ACR) nunca son autenticados. La autenticación se realizará mediante las credenciales utilizadas en el aprovisionamiento. Por favor, verifica en la documentación de _Stratio KEOS_ los repositorios que se soportan en la versión a utilizar.
```

**Review by tperez-stratio — CHANGES_REQUESTED** [review]
@lreciomelero te paso unos cambios. Revisa y aplica si estás conforme. Pide una nueva revisión cuando esté lista para ponerme con la versión en inglés. ¡Gracias!

**Review by tperez-stratio — APPROVED** [review]
@lreciomelero he añadido la versión en inglés. Si ves todo OK mergea. Gracias.

**Pull request merged**